content/guides/next-server-actions.md

```diff
@@ -0,0 +1,137 @@
+
+To create a Neon project and access it from an Next.js application:
+
+- [Create a Neon project](#create-a-neon-project)
+- [Create a Next.js project and add dependencies](#create-a-nextjs-project-and-add-dependencies)
+- [Store your Neon credentials](#store-your-neon-credentials)
+- [Create a Form with Server Actions](#create-a-form-with-server-actions)
+- [Implement Next.js Server Actions](#implement-nextjs-server-actions)
+- [Run the app](#run-the-app)
+
+## Create a Neon project
+
+If you do not have one already, create a Neon project. Save your connection details including your password. They are required when defining connection settings.
+
+1. Navigate to the [Projects](https://console.neon.tech/app/projects) page in the Neon Console.
+2. Click **New Project**.
+3. Specify your project settings and click **Create Project**.
+
+## Create a Next.js project and add dependencies
+
+1. Create an Next.js project if you do not have one. For instructions, see [Automatic Installation](https://nextjs.org/docs/getting-started/installation#automatic-installation) in the Next.js documentation.
+
+2. Add project dependencies using one of the following commands:
+
+    <CodeTabs labels={["node-postgres", "Neon serverless driver"]}>
+
+      ```shell
+      npm install pg
+      ```
+
+      ```shell
+      npm install @neondatabase/serverless
+      ```
+
+    </CodeTabs>
+
+## Store your Neon credentials
+
+Add a `.env` file to your project directory and add your Neon connection string to it. You can find the connection string for your database in the **Connection Details** widget on the Neon **Dashboard**. For more information, see [Connect from any application](/docs/connect/connect-from-any-app).
+
+```shell shouldWrap
+DATABASE_URL="postgres://[user]:[password]@[neon_hostname]/[dbname]"
+```
+
+## Create a Form with Server Actions
+
+Create a simple form that allows users to input a comment. For now, add an action named `create` that you will create in next step with Next.js server actions.
+
+```tsx
+// File: app/page.tsx
+
+export default function Page() {
+  return (
+    <form action={create}>
+      <input type="text" placeholder="write a comment" name="comment" />
+      <button type="submit">Submit</button>
+    </form>
+  );
+}
+```
+
+## Implement Next.js Server Actions
+
+Now, let's add the server action to insert the data into your Postgres (powered by Neon).
+
+<CodeTabs labels={["node-postgres", "Neon serverless driver"]}>
+
+  ```tsx {3,6-16}
+  // File: app/page.tsx
+
+  import { Client } from "pg";
+
+  export default function Page() {
+    async function create(formData: FormData) {
+      "use server";
+      // Create a client instance using `node-postgres`
+      const client = new Client(`${process.env.POSTGRES_URL}`);
```

Comment:
the latest commit in #1644 replaces this with DATABASE_URL, please have a look at https://github.com/neondatabase/website/pull/1644/commits/9a091ce2b47ed54e00d899365af2e63f2cbd8a12?

Reply:
@rishi-raj-jain ok updated!

Reply:
Awesome!